schemas/questionnaire_v1.json

```diff
@@ -152,6 +152,23 @@
       "additionalProperties": false,
       "minProperties": 1
     },
+    "post_submission": {
+      "type": "object",
+      "properties": {
+        "guidance": {
+          "description": "Guidance displayed after the questionnaire is submitted.",
+          "type": "object",
+          "properties": {
+            "contents": {
+              "$ref": "https://eq.ons.gov.uk/common_definitions.json#/contents"
+            }
+          },
+          "required": ["contents"]
```

Comment:
Also need to add `"additionalProperties": false,` at this level, as you can pass in arbitrary keys at the moment.

Reply:
Good spot, fixed in https://github.com/ONSdigital/eq-questionnaire-validator/pull/106/commits/f71340bc317b9b6eb7df72126a33511e14a32de7